README.md

```diff
@@ -1,5 +1,7 @@
 # uptasticsearch
 
+[![CRAN\_Status\_Badge](http://www.r-pkg.org/badges/version-last-release/uptasticsearch)] [![CRAN\_Download\_Badge](http://cranlogs.r-pkg.org/badges/grand-total/uptasticsearch)]
```

Comment:
Hopefully pkgdown will add the badges to its sidebar; that's the intended feature